src/addons/outputLatex.js

```diff
@@ -609,25 +609,12 @@ MathAtom.MathAtom.prototype.toLatex = function(options) {
 
     }
     if (this.superscript) {
-        let sup = latexify(this, this.superscript, options);
-        if (sup.length === 1) {
-            if (sup === '\u2032') {     // PRIME
```

Comment:
@hletzrin I think we'll want to keep these prime/double-prime character replacements then add the braces outside the `if` statement